examples/access_control_function/README.md

```diff
@@ -0,0 +1,383 @@
+2025-08-27T14:55:36.431+0100 [INFO]  provider.terraform-provider-couchbase-capella: Configured Capella client: tf_rpc=ConfigureProvider @caller=/Applications/gh_2.14.6_macOS_amd64/bin/terraform-provider-couchbase-capella/internal/provider/provider.go:171 @module=couchbase_capella authentication_token="***" tf_provider_addr=hashicorp.com/couchbasecloud/couchbase-capella host=http://cloudapi.dev.nonprod-project-avengers.com success=true tf_req_id=ffffffff-aaaa-1414-eeee-000000000000 timestamp="2025-08-27T14:55:36.430+0100"
+couchbase-capella_access_control_function.example_access_function: Refreshing state...
+2025-08-27T14:55:36.435+0100 [WARN]  unexpected data: registry.terraform.io/couchbasecloud/couchbase-capella:stdout=""
+2025-08-27T14:55:37.111+0100 [INFO]  provider: plugin process exited: plugin=/Applications/gh_2.14.6_macOS_amd64/bin/terraform-provider-couchbase-capella/bin/terraform-provider-couchbase-capella id=34382
+2025-08-27T14:55:37.112+0100 [INFO]  provider: configuring client automatic mTLS
+2025-08-27T14:55:37.142+0100 [INFO]  provider.terraform-provider-couchbase-capella: configuring server automatic mTLS: timestamp="2025-08-27T14:55:37.141+0100"
+2025-08-27T14:55:37.157+0100 [INFO]  provider.terraform-provider-couchbase-capella: Configuring the Capella Client: tf_provider_addr=hashicorp.com/couchbasecloud/couchbase-capella tf_req_id=ffffffff-aaaa-1414-eeee-000000000000 tf_rpc=ConfigureProvider @caller=/Applications/gh_2.14.6_macOS_amd64/bin/terraform-provider-couchbase-capella/internal/provider/provider.go:75 @module=couchbase_capella timestamp="2025-08-27T14:55:37.157+0100"
+2025-08-27T14:55:37.157+0100 [INFO]  provider.terraform-provider-couchbase-capella: Configured Capella client: authentication_token="***" host=http://cloudapi.dev.nonprod-project-avengers.com tf_req_id=ffffffff-aaaa-1414-eeee-000000000000 tf_rpc=ConfigureProvider @caller=/Applications/gh_2.14.6_macOS_amd64/bin/terraform-provider-couchbase-capella/internal/provider/provider.go:171 @module=couchbase_capella success=true tf_provider_addr=hashicorp.com/couchbasecloud/couchbase-capella timestamp="2025-08-27T14:55:37.157+0100"
+2025-08-27T14:55:37.160+0100 [INFO]  provider: plugin process exited: plugin=/Applications/gh_2.14.6_macOS_amd64/bin/terraform-provider-couchbase-capella/bin/terraform-provider-couchbase-capella id=34385
+
+Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
+  - destroy
+
+Terraform will perform the following actions:
+
+  # couchbase-capella_access_control_function.example_access_function will be destroyed
+  - resource "couchbase-capella_access_control_function" "example_access_function" {
+      - access_control_function = "function(doc){channel(doc.channels);console.log(doc);}" -> null
+      - app_endpoint_name       = "api" -> null
+      - access_control_function_id          = "ffffffff-aaaa-1414-eeee-000000000000" -> null
```

Comment:
There is no access control function ID, this occurs in all the examples in this readme, preferably redo it with new outputs

Reply:
good catch.  

addressed in https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/390/commits/dadc5b312d0102d23add1002f0780e35c2d7f333